src/stores/ChannelStore.js

```diff
@@ -0,0 +1,44 @@
+const DataStore = require('./DataStore');
+const DMChannel = require('../structures/DMChannel');
+const GroupDMChannel = require('../structures/GroupDMChannel');
+const Constants = require('../util/Constants');
+
+class ChannelStore extends DataStore {
+  create(data, guild, cache = true) {
+    super.create();
```

Comment:
DataStore.create returns undefined (and really does nothing). If you were abstracting some of or all of the creation/removal then this would be great. But as it stands, this is unnecessary.
I will refer to the rest as DATASTORE_CREATE_REMOVE_SUPER